src/inc/clrnt.h

```diff
@@ -853,6 +853,35 @@ typedef union _UNWIND_CODE {
     USHORT FrameOffset;
 } UNWIND_CODE, *PUNWIND_CODE;
 
+// CFI_CODE is 8 byte struct regardless of operation for simplicity.
```

Comment:
You should write this comment to describe more about what a "CFI code" is, and how it is used. Maybe reference a spec. Etc.